app/src/main/java/com/google/android/gms/samples/wallet/SelectBuyFlowActivity.java

```diff
@@ -0,0 +1,110 @@
+import android.view.MenuItem;
+
+import static com.google.android.gms.samples.wallet.BikestoreFragmentActivity.REQUEST_USER_LOGIN;
+
+public class SelectBuyFlowActivity extends AppCompatActivity {
+
+    private DrawerLayout mDrawerLayout;
+    private SelectBuyFlowActivity context = this;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_select_buy_flow);
+        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+        NavigationView navigationView = (NavigationView) findViewById(R.id.nav_view);
+        mDrawerLayout = (DrawerLayout) findViewById(R.id.drawer);
+        ActionBar supportActionBar = getSupportActionBar();
+        if (supportActionBar != null) {
+            VectorDrawableCompat indicator
+                    = VectorDrawableCompat.create(getResources(), R.drawable.ic_menu, getTheme());
+            indicator.setTint(ResourcesCompat.getColor(getResources(),R.color.white,getTheme()));
+            supportActionBar.setHomeAsUpIndicator(indicator);
+            supportActionBar.setDisplayHomeAsUpEnabled(true);
+        }
+        navigationView.setNavigationItemSelectedListener(
+        new NavigationView.OnNavigationItemSelectedListener() {
+            @Override
+            public boolean onNavigationItemSelected(MenuItem menuItem) {
+                menuItem.setChecked(true);
+                if(menuItem.getItemId() == R.id.menu_option_two_step_buyflow){
+                    Intent intent = new Intent(context, ItemListActivity.class);
+                    startActivity(intent);
+                }
+                // TODO: handle navigation to one step and product screen buyflows
+                mDrawerLayout.closeDrawers();
+                return true;
+            }
+        });
+    }
+
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        super.onCreateOptionsMenu(menu);
+        MenuInflater inflater = getMenuInflater();
+        inflater.inflate(R.menu.menu_main, menu);
+        if (((BikestoreApplication) getApplication()).isLoggedIn()) {
+            inflater.inflate(R.menu.menu_logout, menu);
+        } else {
+            inflater.inflate(R.menu.menu_login, menu);
+        }
+        return true;
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        int id = item.getItemId();
+        switch (id) {
+            case android.R.id.home:
+                mDrawerLayout.openDrawer(GravityCompat.START);
+                return super.onOptionsItemSelected(item);
+            case R.id.action_settings:
+                return true;
+            case R.id.login:
+                Intent loginIntent = new Intent(this, LoginActivity.class);
+                loginIntent.putExtra(LoginActivity.EXTRA_ACTION, LoginActivity.Action.LOGIN);
+                startActivityForResult(loginIntent, REQUEST_USER_LOGIN);
```

Comment:
Good point; I copied over code from BikestoreFragmentActivity, but neglected to pull the relevant onActivityResult code.  I'll fix that in the next commit.